From c2a8a0e1d59672cde78f2e85f1e53bfd5869101c Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 15 Jan 2013 04:12:53 +0100
Subject: [PATCH] man: link in wiki documentation about network.target (cherry
 picked from commit aa6c53bf05f27bb094735e7bc2bbfc73ac7bc2ba)

---
 man/systemd.special.xml | 10 ++++++++++
 units/network.target    |  1 +
 2 files changed, 11 insertions(+)

diff --git a/man/systemd.special.xml b/man/systemd.special.xml
index d3b6181..20f1c1f 100644
--- a/man/systemd.special.xml
+++ b/man/systemd.special.xml
@@ -414,6 +414,16 @@
                                         referring to the
                                         <literal>$network</literal>
                                         facility.</para>
+
+                                        <para>This unit is supposed to
+                                        indicate when the network is
+                                        "up", but it is only very
+                                        losely defined what that is
+                                        supposed to mean. Also see <ulink
+                                        url="http://www.freedesktop.org/wiki/Software/systemd/NetworkTarget">Running
+                                        Services After the Network is
+                                        up</ulink> for more
+                                        information.</para>
                                 </listitem>
                         </varlistentry>
                         <varlistentry>
diff --git a/units/network.target b/units/network.target
index 5406f4e..65fc64b 100644
--- a/units/network.target
+++ b/units/network.target
@@ -8,3 +8,4 @@
 [Unit]
 Description=Network
 Documentation=man:systemd.special(7)
+Documentation=http://www.freedesktop.org/wiki/Software/systemd/NetworkTarget
